scripts/protocgen.sh

```diff
@@ -14,17 +14,31 @@ protoc_gen_gocosmos() {
   go get github.com/cosmos/cosmos-sdk@v0.45.6 2>/dev/null
 }
 
-# refresh existing proto files
-rm -rf nibiru/proto proto/
-mkdir -p nibiru/proto/
-cp -r ../nibiru/proto/ proto/
+echo "refresh existing proto files"
+if [ $(basename $(pwd)) = nibiru-py ]
```

Comment:
I'm using it as an indirect form of documentation. It's a bit hard to follow this script because the the `nibiru` repo is assumed to be at `../nibiru`, while the `nibiru` package is `nibiru` and the script actually `cd`s around. 
My intention here was to hint at where the execution was taking place. 

We should probably add an error message for when the nibiru repo isn't found as well to make the dev exp. a little better.